COVERAGE.md

```diff
@@ -30,7 +30,7 @@ We currently cover the following components:
     -   [N/A] Divider
     -   [] Drawer
     -   [X] Dropdown
-    -   [] Field
+    -   [x] Field
```

Comment:
Could you please also update the index.ts file?

Reply:
done